### PR TITLE
Twitter share feature added to Header component. Google Analytics fun…

### PR DESCRIPTION
**Pull request by NickEngmann**
React-share added
Header added to App.js
Twitter Share element added to Header
Google Analytics embedded inside twitter share

### CODE REVIEW

**Comment by nlaz** [comment]
@NickEngmann Nice work. How difficult would it be to add the facebook sharing button? Looks like `react-share` would make it easy? Can you also update the style/location to match the following design?

scratch

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  render() {
+    return (
+      <div className="container mt-3">
+        <div className="row">
+          <ul className=" header-right text-md-right">
+            <li className="list-inline-item">
+              <TwitterShareButton
+                url="http://nlaz.github.io/flashcards-for-developers/#/"
+                title={title}
+                className="Demo__some-network__share-button"
```
Comment (nlaz) [inline src/components/Header.js]: This className isn't used. You can 🔪 it.

File: src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  };
+
+  render() {
+    return (
+      <div className="container mt-3">
+        <div className="row">
+          <ul className=" header-right text-md-right">
+            <li className="list-inline-item">
+              <TwitterShareButton
+                url="http://nlaz.github.io/flashcards-for-developers/#/"
```
Comment (nlaz) [inline src/components/Header.js]: Should we add a task for updating the domain name?

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import { TwitterShareButton, TwitterIcon } from "react-share";
+import * as analytics from "./GoogleAnalytics";
+
+const title = "Helpful Flashcards for Developers @ ";
```
Comment (nlaz) [inline src/components/Header.js]: Is this copy final? Can we think of something creative for sharing?

Comment (NickEngmann) [reply src/components/Header.js]: Sure thing

File: src/components/Header.js (hunk 1 above)
Comment (NickEngmann) [reply src/components/Header.js]: done and updated

File: src/components/Header.js (hunk 2 above)
Comment (NickEngmann) [reply src/components/Header.js]: think that she be done after the name is settled on

**Comment by NickEngmann** [comment]
Forgot to assign myself on the pull request so sorry about being late to updating these changes

**Comment by nlaz** [comment]
Looks good to me. I edited some of the spacing and will merge it now.

**Pull request merged**